contacts.json

```diff
@@ -0,0 +1,302 @@
+      "value": ""
+    },
+    "hs_email_last_send_date": {
+      "value": "2018-08-15T00:01:57.000000Z"
+    },
+    "hs_email_optout_4665933": {
+      "value": ""
+    },
+    "hs_email_last_email_name": {
+      "value": "NEA 2nd Email Campaign 2018 - UR Academy (KO)"
+    },
+    "hs_ip_timezone": {
+      "value": "asia_slash_seoul"
+    },
+    "hs_email_first_send_date": {
+      "value": "2017-06-09T06:05:36.000000Z"
+    },
+    "first_conversion_date": {
+      "value": "2017-04-04T00:11:40.050000Z"
+    },
+    "ip_zipcode": {
+      "value": ""
+    },
+    "hs_analytics_average_page_views": {
+      "value": 2.0
+    },
+    "hs_is_contact": {
+      "value": true
+    },
+    "num_conversion_events": {
+      "value": 1.0
+    },
+    "hs_object_id": {
+      "value": 1247001.0
+    },
+    "ip_latlon": {
+      "value": ""
+    },
+    "hs_email_open": {
+      "value": 16.0
+    },
+    "hs_email_optout_4979379": {
+      "value": ""
+    },
+    "hs_email_optout_4805873": {
+      "value": ""
+    },
+    "hs_email_first_click_date": {
+      "value": "2017-06-09T06:34:47.630000Z"
+    },
+    "firstname": {
+      "value": "Steve Han"
+    },
+    "ip_state": {
+      "value": "seoul teugbyeol-si"
+    },
+    "ip_country": {
+      "value": "south korea"
+    },
+    "hs_email_optout_4460089": {
+      "value": ""
+    },
+    "hs_email_optout_4931834": {
+      "value": ""
+    },
+    "hs_analytics_last_timestamp": {
+      "value": "2019-06-19T12:34:52.512000Z"
+    },
+    "hs_analytics_num_visits": {
+      "value": 11.0
+    },
+    "hs_social_linkedin_clicks": {
+      "value": 0.0
+    },
+    "hs_analytics_last_visit_timestamp": {
+      "value": "2019-06-19T12:17:48.367000Z"
+    },
+    "company": {
+      "value": "Lean Design Asia Inc"
+    },
+    "ip_country_code": {
+      "value": "kr"
+    },
+    "hs_analytics_first_visit_timestamp": {
+      "value": "2016-11-22T14:51:17.884000Z"
+    },
+    "lastmodifieddate": {
+      "value": "2020-01-02T17:19:28.153000Z"
+    },
+    "hs_social_google_plus_clicks": {
+      "value": 0.0
+    },
+    "hs_calculated_form_submissions": {
+      "value": "8a208566-2d9b-40d9-82c8-7e994dc39d69:4792636245:1491264700050"
+    },
+    "hs_analytics_last_referrer": {
+      "value": "https://www.universal-robots.com/career/"
+    },
+    "hs_email_delivered": {
+      "value": 28.0
+    },
+    "hs_social_facebook_clicks": {
+      "value": 0.0
+    },
+    "hs_predictivescoringtier": {
+      "value": "tier_2"
+    },
+    "recent_conversion_date": {
+      "value": "2017-04-04T00:11:40.050000Z"
+    },
+    "hs_email_optout_4478314": {
+      "value": ""
+    },
+    "hs_analytics_source_data_2": {
+      "value": "smartmachinesandfactories.com/"
+    },
+    "hs_email_optout_4434256": {
+      "value": ""
+    },
+    "hs_analytics_source_data_1": {
+      "value": "smartmachinesandfactories.com"
+    },
+    "lifecyclestage": {
+      "value": "subscriber"
+    }
+  }
+}
```

Comment:
`companies.json`, `contacts.json` and `deals.json` shouldn't be included in the tap, please remove them in your next commit